Include/internal/pycore_runtime.h

```diff
@@ -165,7 +165,6 @@ typedef struct pyruntimestate {
 
     /* Is Python fully initialized? Set to 1 by Py_Initialize() */
     int initialized;
```

Comment:
```suggestion
    int initialized;

```

(Restore the removed empty line.)